src/utils.ts

```diff
@@ -330,6 +330,33 @@ export function formatMessage(
   } as LocalMessage;
 }
 
+/**
+ * Takes a LocalMessage, parses the dates back to strings,
+ * and converts the message back to a MessageResponse.
+ *
+ * @param {MessageResponse} message `MessageResponse` object
+ */
+export function unformatMessage(message: LocalMessage): MessageResponse {
```

Comment:
Is this really the opposite of `formatMessage` even though it keeps error, status, reaction_groups?

Reply:
1. `reaction_groups` are introduced as null if the data does not exists or they are calculated
2. `status` is added if it does not exist
3. `error` is introduced as null if it does not exist

I have no problem with the logic, just trying to make sure that the function will not be used by integrators as an opposite to `formatMessage`.

Reply:
Hmm, I think you're right. I'll have a look at these on how viable it is to reverse the formatting process like this. Otherwise I might name this differently to avoid confusion, thanks !

Reply:
As decided on Slack, will keep this as `@private` for now and maybe revisit later if we want to make it fully backwards compatible with `formatMessage`.